test/TEST-70-CRYPTENROLL-TPM2/test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="cryptenroll/cryptsetup with TPM2 devices"
+IMAGE_NAME="cryptenrolltpm2"
+TEST_NO_NSPAWN=1
+TEST_REQUIRE_INSTALL_TESTS=0
+
+# shellcheck source=test/test-functions
+. "${TEST_BASE_DIR:?}/test-functions"
+
+command -v swtpm >/dev/null 2>&1 || exit 0
+command -v tpm2_pcrextend >/dev/null 2>&1 || exit 0
+
+test_append_files() {
+    (
+        local workspace="${1:?}"
+
+        install_dmevent
+        generate_module_dependencies
+        inst_binary tpm2_pcrextend
+        # XXX: Needs a better way to install libraries
```

Comment:
the way we handle this is via pkg-config - add the required pc files to the list at https://github.com/systemd/systemd/blob/main/test/test-functions#L1216

you can find the available pc files at https://packages.ubuntu.com/focal/amd64/libtss2-dev/filelist

Reply:
Thanks, that seems to have worked fine.